torchci/scripts/backfillJobs.mjs

```diff
@@ -83,30 +104,38 @@ async function backfillWorkflowJob(
     console.log(`Failed to find job id ${id}: ${error}`);
     console.log(`Marking job id ${id} as incomplete`);
     console.log(`Querying dynamo entry for job id ${id}`);
-    const dynamoEntry = await client.queries.query({
-      sql: {
-        query: `
+    const dynamoEntry = (
+      await client.queries.query({
+        sql: {
+          query: `
 SELECT
     *
 FROM
     workflow_job j
 WHERE
     j.dynamoKey = '${dynamo_key}'
 `,
-      },
-    });
-    if (dynamoEntry.results.length === 0) {
+        },
+      })
+    ).results;
+    const chDynamoEntry = await queryClickhouse(
+      `SELECT * FROM workflow_job j final WHERE j.dynamoKey = '${dynamo_key}'`,
+      {}
+    );
+
+    if (dynamoEntry.length === 0 && chDynamoEntry.length === 0) {
       console.log(`No dynamo entry found for job id ${id}`);
       return;
     }
-    const result = dynamoEntry.results[0];
+    const result =
```

Comment:
Seems like it only reads the rockset results if CH returned nothing. In that case, what about only querying rockset after you see an empty CH result? That would reduce the overall latency here